migration/ubuntu/20-to-24.md

```diff
@@ -0,0 +1,242 @@
+# Migrate to Ubuntu 24.04 on Openstack
+
+This migration assumes that the environment is running on an Openstack environment.
+This mainly applies to the terraform details, so you should be able to follow along with the kubespray steps.
+
```

Comment:
Shall we add a note on that about 50% openstack qoutas needs to be available?

Reply:
Maybe a `Prerequisites` header?

Reply:
Makes sense, I guess either that or that you do one node at a time which will work but will take significantly more time

Reply:
https://github.com/elastisys/compliantkubernetes-kubespray/pull/409/commits/e79e915dc4ade6c119818be3ec1eb7e14e25f700